deploy/operator/base/deployment.yaml

```diff
@@ -22,7 +22,7 @@ spec:
         runAsNonRoot: true
       containers:
         - name: hacbs-jvm-operator
-          image: hacbs-jvm-operator
+          image: hacbs-jvm-operator:next
```

Comment:
I think the CI failure is legit, and is likely caused by this update https://github.com/redhat-appstudio/jvm-build-service/blob/91a173d2a2421b0d9e169b9678044a9558001aae/deploy/openshift-ci.sh#L22